diff --git a/node_modules/electron-redux/es/main.js b/node_modules/electron-redux/es/main.js
index b607c51..f567603 100644
--- a/node_modules/electron-redux/es/main.js
+++ b/node_modules/electron-redux/es/main.js
@@ -7,7 +7,7 @@ import { p as preventDoubleInitialization } from './misc-f1d565fd.js';
 
 const forwardActionToRenderers = (action, options = {}) => {
   if (validateAction(action, options.denyList)) {
-    webContents.getAllWebContents().forEach(contents => {
+    (webContents.getAllWebContents() || []).forEach(contents => {
       // Ignore chromium devtools
       if (contents.getURL().startsWith('devtools://')) return;
       contents.send(IPCEvents.ACTION, action);
@@ -36,7 +36,7 @@ const stateSyncEnhancer = (options = {}) => createStore => {
       const localAction = stopForwarding(action);
       store.dispatch(localAction); // Forward it to all of the other renderers
 
-      webContents.getAllWebContents().forEach(contents => {
+      (webContents.getAllWebContents() || []).forEach(contents => {
         // Ignore the renderer that sent the action and chromium devtools
         if (contents.id !== event.sender.id && !contents.getURL().startsWith('devtools://')) {
           contents.send(IPCEvents.ACTION, localAction);
